ruby/test/test_helper.rb

```diff
@@ -19,3 +19,5 @@
 Dir[File.expand_path('../support/**/*.rb', __FILE__)].sort.each do |file|
   require file
 end
+
+include RedisHelpers
```

Comment:
Please don't include modules in `::Object`, that's a big footgun. 

Include it in the test classes that need it.